tests/python/pants_test/option/test_options.py

```diff
@@ -47,7 +49,37 @@ def subsystem(scope):
   return ScopeInfo(scope, ScopeInfo.SUBSYSTEM)
 
 
-class OptionsTest(unittest.TestCase):
+class OptionsTestBase(unittest.TestCase):
```

Comment:
It looks like this extraction wasn't used... is a file missing here?

Reply:
This is an artifact of an earlier implementation which had a really screwy way to test stderr. This will be reverted.

Reply:
This was reverted as of 338a11c7e565e33db15e7f117f6907683b6fc80c!